server/src/services/organizations.service.ts

```diff
@@ -112,11 +112,62 @@ export class OrganizationsService {
     }, manager);
   }
 
-  async fetchUsers(user: any): Promise<FetchUserResponse[]> {
-    const organizationUsers = await this.organizationUsersRepository.find({
-      where: { organizationId: user.organizationId },
-      relations: ['user'],
+  async fetchUsersByValue(user: any, searchInput: string): Promise<any> {
+    const organizationUsers = await createQueryBuilder(OrganizationUser, 'organization_user')
+      .innerJoinAndSelect('organization_user.user', 'user')
+      .where('organization_user.organization_id = :organizationId', {
+        organizationId: user.organizationId,
+      })
+      .andWhere(
+        new Brackets((qb) => {
+          qb.where('lower(user.email) like :email', {
+            email: `%${searchInput.toLowerCase()}%`,
+          })
+            .orWhere('lower(user.firstName) like :firstName', {
+              firstName: `%${searchInput.toLowerCase()}%`,
+            })
+            .orWhere('lower(user.lastName) like :lastName', { lastName: `%${searchInput.toLowerCase()}%` });
+        })
+      )
```

Comment:
this query is repeating in three functions, fetchUser, fetchUserByValue and usersCount, shall we move the query part to separate function and return query?